index.html

```diff
@@ -221,8 +221,8 @@ <h3>Open Source Diversity<br>Berlin</h3>
 				</a>
 				<a class="col-md-4" href="https://diversifyingopensource.github.io" target="_blank" rel="noopener">
 					<img alt="" src="img/projects/diversifyfoss.png" />
-					<h3>Difersify Open Source<br>Bangalore</h3>
-					<p>We help, encourage and celebrate underrepresented people to contribute in open source projects and speak at conferences!</p>
+					<h3>Diversify Open Source<br>Bangalore</h3>
+					<p>We help encourage, and celebrate underrepresented people to contribute to open source projects and speak at conferences!</p>
```

Comment:
Help and encourage are two different efforts in this particular context. Please separate them with a comma.